### ISSUE TITLE
Dropdown should ignore clicks on <select>s (and <label>s?) within the dropdown

### ISSUE THREAD

**Issue by yksht** [twbs/bootstrap#17656]
Hello.
I tried to use dropdown plugin like a special area for options (some radio buttons) and found a strange behavior. You can see it here (try to click on select tag or labels for radio buttons): http://jsbin.com/jorota/edit?html,output

There is a simple check in bootstrap.js in this plugin
```javascript
/input|textarea/i.test(e.target.tagName)
```

this behavior can be fixed by adding new tags through the '|'
```javascript
/input|textarea|select|label/i.test(e.target.tagName)
```

is it possible to add this fix to release? or this is not a type of regular using of this plugin? but in this case: why there is 'textarea' in condition? :)

thank you :) and sorry for my bad english

**Comment by yksht** [comment]
Hi again :)

this fix was added in version 3.3.5 but was lost it in 3.3.7
are you going to support this in 3.x.x? :)

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
